src/components/ChatApp.css

```diff
@@ -270,7 +270,7 @@ textarea {
 }
 .leaflet-container {
   height: 150px;
-  width: 80%;
+  width: 250px;
```

Comment:
Initially since text was also displayed in the same div, width auto was working good. Now since its the only element, width: 80% is not working. So i tried with vw also, but 250px seemed fine. 
Can suggest if you know a workaround.